tests/framework/statistics/consumer.py

```diff
@@ -77,14 +77,16 @@ def _validate(self):
         """
         for ms_name in self._statistics_defs:
             if ms_name not in self._measurements_defs:
-                return False
```

Comment:
For the current PR this is ok, but I am not sure if this is the best way in terms of extensibility. This function will either return true or assert. There is no case for false. Maybe in the future you will want to reuse it, and this behaviors is quite strict. I would use a print or return a string, where empty string will be ok. This way the upper level function can decide if it wants to assert or not.

I am not blocking the PR on this. Is just food for thought.

Reply:
Another way would be to have an extra configuration to this function -> `assert=True`